spec/system/create_image_spec.rb

```diff
@@ -29,11 +29,9 @@
         attach_file('files[]', upload_file_path)
       end
       click_link 'Descriptions' # switch tab
-      fill_in('Title', with: 'My Test Work')
-      fill_in('Creator', with: 'Doe, Jane')
-      fill_in('Keyword', with: 'testing')
-
-      select('In Copyright', from: 'Rights statement')
+      find('.image_title').set('my title')
```

Comment:
I'd be curious to hear why `fill_in('Title'...` isn't working and was changed on all of these tests?

Reply:
So the issue is that capybara looks for labels, and it does a fuzzy search. Since we have a few different fields with "Title" in the label, it was fiding multiple and I was getting an "Ambiguous match, 3 fields with title that arent disabled" error. So we need to target the title fields better.

Reply:
Thanks, that makes sense!